spec/03-plugins/27-aws-lambda/08-sam-integration_spec.lua

```diff
@@ -0,0 +1,101 @@
+local helpers = require "spec.helpers"
+local sam = require "spec.fixtures.aws-sam"
+local utils = require "spec.helpers.perf.utils"
+
+local sam_describe
+do
+  local arch_type = sam.get_os_architecture()
+  local is_sam_installed, _ = sam.is_sam_installed()
+  if arch_type ~= "aarch64" and is_sam_installed then
+    sam_describe = describe
+  else
+    sam_describe = pending
+  end
+end
+
+if sam.get_os_architecture() ~= "aarch64" then
```

Comment:
Can you add a comment explaining why this is required?

Reply:
Comment added.